### ISSUE TITLE
Forge suport tracking meta-issue

### ISSUE THREAD

**Issue by astei** [VelocityPowered/Velocity#107]
I'd hoped to not open up a tracking issue, but I'm going to do so anyway to help us organize this:

* #78 - Long-standing, has been an issue since we added Forge support.
* Problem with Velocity joining an FTB Continuum server with matching modded client: https://pastebin.com/Jq4Vt0Tw

**Comment by NEXUS2345** [comment]
I've spent the last 2.5 hours attempting to get around the timeout issues and performing the following changes to Velocity and a DW20 1.12.2 SpongeForge server seem to have had an impact:

Increased read timeout threshold on DW20 to 120
Increased timeout thresholds on Velocity x10
Decreased compression threshold on Velocity to match that of DW20 (1024 -> 256)
Removed JourneyMap from DW20
Changed forge advertisement on Velocity to true

I believe the most impact came from decreasing the compression threshold to match that of the server. This is just speculation based on myself working with 5 other players to get this working. Client reinstalls were also done, which seemed to have some impact also.

Hope this helps.
